src/recordlinker/schemas/pii.py

```diff
@@ -332,6 +345,12 @@ def feature_iter(self, feature: Feature) -> typing.Iterator[str]:
             for address in self.address:
                 if address.county:
                     yield address.county
+        elif feature == Feature.DRIVERS_LICENSE:
+            if self.drivers_license:
+                if self.drivers_license.value:
+                    yield self.drivers_license.value
+                if self.drivers_license.authority:
+                    yield self.drivers_license.authority
```

Comment:
Depending on the matching function that's being used, the results could be weird (particularity with the "any" matchers, as we'd just be checking to see if the ID was issued in the same state).  I wonder if it would be better to concat in this case, open to ideas though.

```python
yield f"{self.drivers_license.authority}|{self.drivers_license.value}"
```

Reply:
Oh I see what you're saying. I think concatenating is a good idea. Because we essentially want to make sure the actual license value is the same and the state, not either or. So putting them together in one string makes sense to me

Reply:
I've been thinking about this, esp related to the triplets concept that Matthew was discussing yesterday.  In general I'm wondering if some format of `{id.type}:{id.authority}:{id.value}` will be good for making a generic id that we can compare.  Don't worry about that in this PR, but we might pivot there next week.